unit_test/test_spdm_requester/get_measurement_extension_log.c

```diff
@@ -0,0 +1,799 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "spdm_unit_test.h"
+#include "internal/libspdm_requester_lib.h"
+#include "internal/libspdm_secured_message_lib.h"
+
+#if LIBSPDM_ENABLE_CAPABILITY_MEAS_CAP
+
+uint8_t m_mel_buffer[LIBSPDM_MAX_MEASUREMENT_EXTENSION_LOG_SIZE];
+size_t m_mel_buffer_size;
+
+void test_generate_mel(spdm_mel_entry_dmtf_t * mel_entry,
+                       uint32_t mel_index, uint8_t meas_index,
+                       void * Dmtf_spec_measurement_value,
+                       size_t dmtf_spec_measurement_value_size)
+{
+    mel_entry->mel_index = mel_index;
+    mel_entry->meas_index = meas_index;
+
+    libspdm_write_uint24(mel_entry->reserved, 0);
+    mel_entry->measurement_block_dmtf_header.dmtf_spec_measurement_value_type =
+        SPDM_MEASUREMENT_BLOCK_MEASUREMENT_TYPE_VERSION |
+        SPDM_MEASUREMENT_BLOCK_MEASUREMENT_TYPE_RAW_BIT_STREAM;
+    mel_entry->measurement_block_dmtf_header.dmtf_spec_measurement_value_size =
+        dmtf_spec_measurement_value_size;
+    libspdm_copy_mem((void *)(mel_entry + 1), dmtf_spec_measurement_value_size,
+                     Dmtf_spec_measurement_value, dmtf_spec_measurement_value_size);
+}
+
+/**
+ * @param  measurement_record_length  On input, indicate the size in bytes of the destination buffer to store the measurement record.
+ *                                    On output, indicate the size in bytes of the measurement record.
+ * @param  measurement_record         A pointer to a destination buffer to store the measurement record.
```

Comment:
there is no parameter named `measurement_record_length` and `measurement_record`.